spec/components/layout/footer_component_spec.rb

```diff
@@ -20,4 +20,26 @@
 
     expect(page).not_to be_rendered
   end
+
+  describe "when the cookies consent feature is enabled" do
+    before { Setting["feature.cookies_consent"] = true }
+
+    it "shows a link to the cookies management modal when the cookies consent is enabled" do
+      render_inline Layout::FooterComponent.new
+
+      page.find(".subfooter") do |footer|
+        expect(footer).to have_css "a[data-open=cookies_consent_management]", text: "Manage cookies"
+      end
+    end
+
+    it "does not show a link to the cookies management modal when the cookies consent is disabled" do
+      Setting["feature.cookies_consent"] = false
+
+      render_inline Layout::FooterComponent.new
+
+      page.find(".subfooter") do |footer|
+        expect(footer).not_to have_css "a[data-open=cookies_consent_management]", text: "Manage cookies"
```

Comment:
Since this is a negative expectation, we can make it more generic:

```suggestion
        expect(footer).not_to have_content "Manage cookies"
```